src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,128 @@
+import React,{useState,FC}from 'react'
+import { classNames } from '../../utils/classNames';
+import { uid } from '../../utils/uuid';
+import { IconPath } from './Icons';
+
+export type boxSize = 'ex-small'|'small' | 'normal' | 'large' | 'ex-larger';
+export type iconType = 'home' | 'spinner' | 'angle down' | 'plus' | 'home' | 'users' | 'times' | 'search' | 'star' | 'moon' | 'heart' | 'smile wink' | 'truck' | 'credit card' | 'check';
+export type themeColor = 'primary' | 'secondary' | 'success' | 'info' | 'warning' | 'danger' | 'dark' | 'default';
+
+export interface ICheckboxProps {
+  /** set customized style */
+  className?: string;
+  /** set checkbox size */
+  checkSize?: boxSize
+  /** make checkbox bar always check */
+  checked?: boolean | undefined;
+  /** add specific icon to input */
+  icon?: iconType;
+  /** add specific icon theme to input */
+  iconTheme?: themeColor;
+  /**customize checkbox color */
+  checkBgTheme? : themeColor;
+  //**set background theme when check on check icon */
+  checkedBgTheme? : themeColor;
+  /** pass a callback function out-site of props */
+  onChange?: any;
```

Comment:
onChange should not be of type any.  Otherwise user can pass it other data types like number or undefined, when it is supposed to be taking a function

Reply:
Thanks for pointing that out, I fix it right now